### PR TITLE
SSA: allow testing multiple images on providers

### PR DESCRIPTION
**Pull request by vrutkovs**
{{pytest: cfme/tests/infrastructure/test_instance_analysis.py --use-provider 'rhevm34' -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6d84ad2fda3810d56cd62007a5faab62befc60d4:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_instance_analysis.py:114:5: E265 block comment should start with '# '
./cfme/tests/infrastructure/test_instance_analysis.py:115:5: E265 block comment should start with '# '
./cfme/tests/infrastructure/test_instance_analysis.py:193:20: E702 multiple statements on one line (semicolon)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 423dd79f1534534763ad2f6d19fb6a4c6b022eaf:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f0893dde5127632f8cc06ad25b852731568617fa:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_instance_analysis.py:245:101: E501 line too long (102 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c6d34d7d576f2334377ac51964985b1778b4f417:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_instance_analysis.py:24:12: E241 multiple spaces after ':'
./cfme/tests/infrastructure/test_instance_analysis.py:27:12: E241 multiple spaces after ':'
./cfme/tests/infrastructure/test_instance_analysis.py:242:5: E265 block comment should start with '# '
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f3e52bcc3a983aef2448ba59d64043641276b2fa:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0c8ef58aecf862108ccdb89964bbbf28ad65e658:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b96525335fb8d343b58112206ce9e6bcc7b4bbc0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a9f5580f17180e9c0d76d4e49807b75353c01fd8:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./fixtures/soft_assert.py:223:28: W292 no newline at end of file
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4fac735eefd46098642ef6c5caa04e65b2ccfee2:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./fixtures/soft_assert.py:223:28: W292 no newline at end of file
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6c14e96f40fccda82b8a4badd5a60a50df1058f4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 90ac28361d1247ec826899ba7e3068849d2b261a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c73e6d04cfcb59e15f0e94b6f45105b113cbd388:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5a9af82bd1dc9593030ac485a73a434edf1191b0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fb4194d6edd9ad8ebd7872f6ac46ab979771dbec:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8ba96733820e3b0fa17839b91a74fa85003f265e:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_instance_analysis.py:392:8: E713 test for membership should be 'not in'
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3274426c6702b52f9cdd3526e860396e589d5e1b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 020245533d7ce1c002bc56e71a8062a90a00b4a8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ec4a8dcf826322505c148c104af1ccc2b2b11f01:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/infrastructure/test_instance_analysis.py [hunk 1]
```diff
@@ -9,51 +9,100 @@
...
+        'install-command': 'env DEBIAN_FRONTEND=noninteractive apt-get -y install {}',
+        'package-number': 'dpkg --get-selections | wc -l'},
+}
 
 
 def pytest_generate_tests(metafunc):
     # Filter out providers without templates defined
-    argnames, argvalues, idlist = testgen.infra_providers(metafunc, 'provisioning')
+    argnames, argvalues, idlist = testgen.infra_providers(
+        metafunc, "vm_analysis_new", require_fields=True)
```
Comment (mfalesni) [inline cfme/tests/infrastructure/test_instance_analysis.py]: Just fyi: https://github.com/RedHatQE/cfme_tests/blob/9d16d76e4ed86b214232aec7db2b2b9a911853db/utils/testgen.py#L331

`require_fields=` is True by default but that is not an issue, gonna merge it anyway.

**Comment by dajoRH** [comment]
Lint report for commit 1c6fd7d4d59d260bcefb111155ae544eb5cbd3bf:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fe9324bd528d20fb6f3cc047adb332128548b21e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0efbc1b8b54eb1bba755d979594ebb3ce5934a48:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1c5ca178c767b810d3384996935c5e6f9d65b41d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4fc546e3128a69d3f283c5649513f4cde5b1861e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8a8192cffd45cff60f2769380704942891d6325c:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_instance_analysis.py:107:101: E501 line too long (107 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 177a58e6a7f8e424cf85924c022e051d97c8cbc3:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_instance_analysis.py:107:101: E501 line too long (107 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f33dbbdbd145ca17620948f3d23920ca77aa139d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a6b4241c3825ea5a465116d037c6cbaf868ed83b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fda00c6690f8c8bc18ecf2a4ce278d9c637c4721:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d19286cb49f242fdf12537564ce8f924aaffb3b2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 35d86934d674b7adab8a6c3df733148f7e139555:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 24d1ac46d178fd45a5cbb3bbeb586a83e7a54e1f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a7c15f19522aa67441b77f9444a98668093c8651:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7d43de2890b04d0bbf9749dbd52965b4b2958e66:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 656aa95a4fc3e0cd820c580d3c00d4f3cb346d05:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3ef0c71be9f749f5351b85c3169b7c747a69223f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bb3dd3bd6c806ff235397f83d821e1967c8ace45:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**